<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>conditionDialog</class>
 <widget class="QDialog" name="conditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>83</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Condition</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/main/call_split-24px.svg</normaloff>:/main/call_split-24px.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="conditionIconLabel">
       <property name="maximumSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/main/call_split-24px.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="conditionNameLabel">
       <property name="text">
        <string>Condition Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="conditionName">
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="conditionDelete">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/main/delete-24px.svg</normaloff>:/main/delete-24px.svg</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="conditionLabel">
     <property name="text">
      <string>Condition</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="processPathBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="processPath">
     <property name="placeholderText">
      <string>Executable to launch</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="conditionRunning">
     <item>
      <property name="text">
       <string>is running</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>is NOT running</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="processLabel">
     <property name="text">
      <string>Executable Path</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="statusBarLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>conditionName</tabstop>
  <tabstop>processPath</tabstop>
  <tabstop>conditionRunning</tabstop>
  <tabstop>conditionDelete</tabstop>
  <tabstop>processPathBtn</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
